src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -105,20 +105,22 @@ describe('VAOS <DateTimeSelectPage>', () => {
     ).to.be.ok;
 
     // it should display 2 calendar months for VA appointments
-    expect(
-      screen.getByRole('heading', {
-        level: 2,
-        name: moment().format('MMMM YYYY'),
-      }),
-    ).to.be.ok;
-    expect(
-      screen.getByRole('heading', {
-        level: 2,
-        name: moment()
-          .add(1, 'M')
-          .format('MMMM YYYY'),
-      }),
-    ).to.be.ok;
+    await waitFor(() => {
```

Comment:
You don't need to use `waitFor` here, you can do `await screen.findByRole(` in both of those `expect` statements.

Reply:
I tried that and it still failed. I had the await on the 1st heading and not the 2nd with the assumption that if the 1st await succeeds, the 2nd should be good.